src/main_thread/api/debug/modules/segment_buffer_content.ts

```diff
@@ -75,13 +79,29 @@ export default function createSegmentSinkGraph(
 
   function updateBufferMetrics() {
     const showAllInfo = isExtendedMode(parentElt);
-    const inventory = bufferMetrics?.segmentSinks[bufferType].segmentInventory;
+    const metricsForType = bufferMetrics?.segmentSinks[bufferType];
+    const inventory = metricsForType?.segmentInventory;
     if (bufferMetrics === null || inventory === undefined) {
       bufferGraphWrapper.style.display = "none";
+      bufferSizeElt.innerHTML = "";
       currentRangeRepInfoElt.innerHTML = "";
       loadingRangeRepInfoElt.innerHTML = "";
     } else {
       bufferGraphWrapper.style.display = "block";
+      if (metricsForType?.sizeEstimate !== undefined) {
+        const sizeEstimate = metricsForType.sizeEstimate;
+        let sizeStr: string;
+        if (sizeEstimate > 2e6) {
+          sizeStr = (sizeEstimate / 1e6).toFixed(3) + "MB";
```

Comment:
3 digits after the "." is a bit to precise to me. I wouldn't mind having 1 or 2.